src/handlers/messages-get.ts

```diff
@@ -68,18 +65,10 @@ export class MessagesGetHandler implements MethodHandler {
       }
```

Comment:
Nit: There's a comment a few lines above here that says `! NOTE: this is somewhat duplicate code that also exists in StorageController.query.`. Do you mind removing that since it's not relevant anymore?